testing/sakura/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Niko Dittmann <mail@niko-dittmann.com>
+# Maintainer: Niko Dittmann <mail@niko-dittmann.com>
+pkgname=sakura
+pkgver="3.5.0"
+pkgrel=0
+pkgdesc="Sakura Terminal Emulator"
+url="https://launchpad.net/sakura"
+arch="all"
+license="GPL-2.0"
+depends="vte3 pango gtk+3.0 glib libintl"
+makedepends="cmake vte3-dev pango-dev gtk+3.0-dev glib-dev libintl"
+install=""
+subpackages="$pkgname-doc"
+source="https://launchpad.net/sakura/trunk/$pkgver/+download/$pkgname-$pkgver.tar.bz2"
+builddir="$srcdir/"
+
+build() {
+	cd "$builddir/$pkgname-$pkgver"
+	cmake -DCMAKE_INSTALL_PREFIX=/usr .
+	make
+}
+
+package() {
+	cd "$builddir/$pkgname-$pkgver"
+	make DESTDIR="$pkgdir" install
+}
+
+check() {
+        sakura --version 2>&1 | grep $pkgver
```

Comment:
This is not a valid check() function.
Additionally it will fail always on builders as there is no `sakura` binary in path.
If there is no test code available you could set the variable `options="!check"` and remove the `check()` function.